web/src/compose_tooltips.ts

```diff
@@ -209,18 +209,44 @@ export function initialize(): void {
     });
 
     tippy.delegate("body", {
```

Comment:
Moved to using a single tippy instance here since I don't think tippy was handling two instances on a single element well.